src/main/resources/hudson/plugins/ec2/EC2Computer/configure.jelly

```diff
@@ -121,11 +121,10 @@ THE SOFTWARE.
     	</f:entry>
 
         <f:dropdownList name="amiType" title="${%AMI Type}">
-            <f:dropdownListBlock value="0" title="${instance.amiType.descriptor.displayName}" selected="true">
-		        <j:set var="instance" value="${instance.amiType}" />
-                <tr><td><input type="hidden" name="stapler-class" value="${instance.descriptor.clazz.name}"/></td></tr>
-                <st:include from="${instance.descriptor}" page="${instance.descriptor.configPage}" />
-            </f:dropdownListBlock>
+          <f:dropdownListBlock value="0" title="${instance.amiType.descriptor.displayName}" staperClass="${instance.descriptor.clazz.name}" selected="true">
```

Comment:
would [form:dropdownDescriptorSelector](https://reports.jenkins.io/core-taglib/jelly-taglib-ref.html#form:dropdownDescriptorSelector) do all of this in one shot?
[code](https://github.com/jenkinsci/jenkins/blob/53f300d5ec07eb5efa3774d3f12455615d2f3450/core/src/main/resources/lib/form/dropdownDescriptorSelector.jelly)